files/standard.json

```diff
@@ -3,10 +3,10 @@
     "roomId": 0,
     "cameras": [
       {
-        "link": "files/postit.mov"
+        "link": "files/1_20180522_114113.h264"
```

Comment:
This should not be pushed to master as these stream movies are not in the files.
Maybe create a separate json file for the streams so it will be easier for us to load that in, or just let everyone store their own copy.